src/wp-includes/class-wp-user.php

```diff
@@ -722,6 +742,7 @@ public function add_cap( $cap, $grant = true ) {
 	 * @param string $cap Capability name.
 	 */
 	public function remove_cap( $cap ) {
+		$this->load_capablity_data();
```

Comment:
This feedback is not directly related to this line, but it looks like you're not loading the capability data in the `remove_all_caps()` method below. This could lead to unexpected consequences I think. It would be safer to add it there too.

Reply:
Fixed.